tests/Behat/Context/FixtureContext.php

```diff
@@ -4,14 +4,11 @@
 use DNADesign\Elemental\Models\BaseElement;
 use DNADesign\Elemental\Models\ElementalArea;
 use DNADesign\Elemental\Models\ElementContent;
-use SilverStripe\BehatExtension\Context\FixtureContext as BaseFixtureContext;
+use SilverStripe\CMS\Tests\Behaviour\FixtureContext as BaseFixtureContext;
```

Comment:
Necessary to include the anchor dropdown fixture method - we can't just add the `SilverStripe\CMS\Tests\Behaviour\FixtureContext` class to the fixtures in `behat.yml` because it has the same superclass as this one, which causes behat to fail since it doesn't know whether to call methods from this fixture class or the CMS one.
Using the CMS fixture class as the superclass here resolves that problem.